store/store.go

```diff
@@ -274,7 +274,9 @@ type Store struct {
 
 	// Latest log entry index actually reflected by the FSM. Due to Raft code
 	// this value is not updated after a Snapshot-restore.
-	fsmIdx *atomic.Uint64
+	fsmIdx         *atomic.Uint64
+	fsmUpdateTime  *AtomicTime
```

Comment:
Add a comment to specify that this is the node-local time?

Reply:
OK